tests/test_model_derivatives.py

```diff
@@ -163,19 +163,27 @@ def f(value):
         h3 = model.H3.value
         stepgen = numdifftools.MaxStepGenerator(abs(h3) / 2)
     elif param == "FB0":
-        stepgen = numdifftools.MaxStepGenerator(np.abs(model.FB0.value) * 1e-2)
+        stepgen = numdifftools.MaxStepGenerator(
+            np.abs(model.FB0.value.astype(np.float64)) * 1e-2
```

Comment:
I missed the rational for all of these explicit casts to float64.  What was the reason?  As I mentioned, I think the FB terms (of which there are often many, like 10+ for redbacks) are highly numerically unstable.  Do we need to do this for some reason?

Reply:
The numerical derivative routines in `numdifftools` rely on `numdifftools.extrapolation.convolve1d` which calls `scipy.ndimage.correlate1d`.  Those routines fail explicitly with an error about not recognizing `float128`.  The last release of `numdifftools` was 2015, so I don't think it will be updated.  

The casting is not in the actual calculation, but only in the tests which compare our analytical derivatives to the numerical values.  So I don't think this will affect our results.